azure-stack/partner/azure-stack-vaas-release-notes.md

```diff
@@ -24,6 +24,28 @@ ms.lastreviewed: 10/28/2019
 
 This article has the release notes for Azure Stack Validation as a Service.
 
+## Version 5.1.49
```

Comment:
Service release versions are added to the documentation. The current build of the service is 4.3.5.3.